docs/designers-developers/developers/block-api/block-registration.md

```diff
@@ -311,6 +311,40 @@ transforms: {
 ```
 {% end %}
 
+In addition to accepting an array of known block types, the `blocks` option also accepts a "wildcard" (`"*"`). This allows for transformations which apply to _all_ block types (eg: all blocks can transform into `core/group`):
+
+{% codetabs %}
+{% ES5 %}
+```js
+transforms: {
+    from: [
+        {
+            type: 'block',
+            blocks: [ '*' ], // wildcard - match any block
+            transform: function( attributes, innerBlocks ) {
```

Comment:
To confirm, we're addressing documentation consistency in a follow-up task? Noting the difference in what's documented here as conflicting with the new proposed API for transform `convert` function.

Reply:
Yes. Riad suggested a separate task Pr to update the docs. Esp. given this is a soft deprecation and is opt in.